src/onboarding/Setup/Setup.js

```diff
@@ -2,13 +2,13 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { useViewport, GU } from '@aragon/ui'
 import Templates from '../Templates/Templates'
-import ConfigureStepsPanel from './ConfigureStepsPanel'
+import SetupStepsPanel from './SetupStepsPanel'
 import ConfigureTemplateScreens from './ConfigureTemplateScreens'
 import * as CreateStatuses from '../Create/create-statuses'
 import { OrgTemplateType } from '../../prop-types'
 
-export const CONFIGURE_MODE_SELECT = Symbol('CONFIGURE_MODE_SELECT')
-export const CONFIGURE_MODE_CONFIGURE = Symbol('CONFIGURE_MODE_CONFIGURE')
+export const SETUP_MODE_SELECT = Symbol('SETUP_MODE_SELECT')
+export const SETUP_MODE_CONFIGURE = Symbol('SETUP_MODE_CONFIGURE')
```

Comment:
Let’s alphabetize these as well. :)

Reply:
Going to leave this order as-is—the natural order for the user (and for the code) is this order.